src/ConductorSharp.Engine/Builders/WorkflowDefinitionBuilder.cs

```diff
@@ -156,6 +156,11 @@ string script
             Expression<Func<TWorkflow, F>> input
         ) where F : IRequest<G> => AddAndReturnBuilder(new JsonJqTransformTaskBuilder<F, G>(refference.Body, input.Body));
 
+        public ITaskOptionsBuilder AddTask<F, G>(
```

Comment:
Let's try giving these type parameters meaningful names for new code. I suggest `TInput `and `TOutput`

Reply:
Agreed, but i suggest doing this revamp in a different PR where we tidy up these things